mail_restrict_follower_selection/README.rst

```diff
@@ -7,6 +7,8 @@ Restrict follower selection
 
 This module was written to allow you to restrict the selection of possible followers. For example, if you use the social ERP functions only internally, it makes sense to filter possible followers for being employees. Otherwise, you'll get a quite crowded list of partners to choose from.
 
+Moreover, the module disables the option to automatically add followers that do not meet the domain.
+
```

Comment:
I'm missing here the same explanation in Configuration section as in the v8 version:

https://github.com/OCA/social/blob/e640e21bcc3f35a9b45ae97843f0540b505bcea0/mail_restrict_follower_selection/README.rst

Reply:
Sorry, I didn't see the text. Yes, it's OK.